src/channels-api.js

```diff
@@ -386,23 +386,23 @@ class ChannelsApi implements IStreamingAPI {
     }
   }
 
-  request(stream: string, payload: Object, requestId: string=UUID.generate()): Promise<Object> {
+  request(stream: string, payload: Object, request_id: string=UUID.generate()): Promise<Object> {
```

Comment:
Would you keep this as `requestId` here, but use `{request_id: requestId}` in the call to `_buildMultiplexedMessage`?